[2024-09-30T22:07:30.331+0000] {processor.py:186} INFO - Started process (PID=60) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:07:30.344+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:07:30.348+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:07:30.347+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:07:32.600+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:07:32.108+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:07:32.606+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:07:32.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 2.484 seconds
[2024-09-30T22:08:29.274+0000] {processor.py:186} INFO - Started process (PID=67) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:08:29.276+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:08:29.279+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:08:29.278+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:08:29.825+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:08:29.798+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:08:29.832+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:08:29.880+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.619 seconds
[2024-09-30T22:09:24.000+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:09:24.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:09:24.003+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:09:24.003+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:09:24.341+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:09:24.333+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:09:24.343+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:09:24.389+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.400 seconds
[2024-09-30T22:10:20.928+0000] {processor.py:186} INFO - Started process (PID=77) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:10:20.942+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:10:20.957+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:10:20.956+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:10:22.069+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:10:22.033+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:10:22.073+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:10:22.138+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 1.240 seconds
[2024-09-30T22:10:59.520+0000] {processor.py:186} INFO - Started process (PID=82) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:10:59.524+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:10:59.528+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:10:59.527+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:11:00.724+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:11:00.706+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:11:00.732+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:11:00.897+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 1.398 seconds
[2024-09-30T22:11:53.807+0000] {processor.py:186} INFO - Started process (PID=87) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:11:53.809+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:11:53.812+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:11:53.812+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:11:54.190+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:11:54.185+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:11:54.195+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:11:54.242+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.448 seconds
[2024-09-30T22:12:45.760+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:12:45.761+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:12:45.764+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:12:45.763+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:12:46.168+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:12:46.160+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:12:46.170+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:12:46.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.486 seconds
[2024-09-30T22:13:41.433+0000] {processor.py:186} INFO - Started process (PID=102) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:13:41.435+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:13:41.437+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:13:41.437+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:13:41.944+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:13:41.939+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:13:41.945+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:13:42.037+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.613 seconds
[2024-09-30T22:14:37.033+0000] {processor.py:186} INFO - Started process (PID=112) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:14:37.034+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:14:37.037+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:14:37.037+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:14:37.427+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:14:37.423+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:14:37.429+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:14:37.481+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.455 seconds
[2024-09-30T22:15:35.974+0000] {processor.py:186} INFO - Started process (PID=253) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:15:35.976+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:15:35.990+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:15:35.989+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:15:36.941+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:15:36.921+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:15:36.951+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:15:37.038+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 1.096 seconds
[2024-09-30T22:16:31.408+0000] {processor.py:186} INFO - Started process (PID=258) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:16:31.412+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:16:31.414+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:16:31.414+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:16:32.310+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:16:32.221+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:16:32.331+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:16:32.514+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 1.113 seconds
[2024-09-30T22:17:25.711+0000] {processor.py:186} INFO - Started process (PID=263) to work on /opt/airflow/dags/Load.py
[2024-09-30T22:17:25.722+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T22:17:25.732+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:17:25.731+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T22:17:26.274+0000] {logging_mixin.py:190} INFO - [2024-09-30T22:17:26.264+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T22:17:26.280+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T22:17:26.530+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.845 seconds
[2024-09-30T23:19:42.080+0000] {processor.py:186} INFO - Started process (PID=48) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:19:42.083+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:19:42.087+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:19:42.086+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:19:42.613+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:19:42.605+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:19:42.617+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:19:42.682+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.626 seconds
[2024-09-30T23:20:38.909+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:20:38.912+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:20:38.923+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:20:38.922+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:20:39.295+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:20:39.287+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:20:39.299+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:20:39.362+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.468 seconds
[2024-09-30T23:22:15.316+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:22:15.324+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:22:15.336+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:22:15.335+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:22:16.329+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:22:15.995+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:22:16.397+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:22:16.593+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 1.300 seconds
[2024-09-30T23:22:51.614+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:22:51.617+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:22:51.630+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:22:51.629+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:22:52.143+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:22:52.136+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:22:52.148+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:22:52.209+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.628 seconds
[2024-09-30T23:33:40.390+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:33:40.392+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:33:40.396+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:33:40.396+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:33:40.694+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:33:40.686+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:33:40.700+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:33:40.773+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.395 seconds
[2024-09-30T23:34:32.342+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:34:32.343+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:34:32.346+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:34:32.346+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:34:32.496+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:34:32.490+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:34:32.499+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:34:32.552+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.216 seconds
[2024-09-30T23:35:32.040+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:35:32.042+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:35:32.046+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:35:32.046+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:35:32.184+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:35:32.179+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:35:32.186+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:35:32.227+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.195 seconds
[2024-09-30T23:36:25.100+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:36:25.103+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:36:25.113+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:36:25.111+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:36:25.606+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:36:25.547+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:36:25.709+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:36:25.803+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.717 seconds
[2024-09-30T23:37:49.092+0000] {processor.py:186} INFO - Started process (PID=58) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:37:49.093+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:37:49.096+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:37:49.095+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:37:49.217+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:37:49.211+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:37:49.219+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:37:49.263+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.178 seconds
[2024-09-30T23:38:44.870+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:38:44.871+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:38:44.874+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:38:44.874+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:38:44.989+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:38:44.983+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:38:44.991+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:38:45.036+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.173 seconds
[2024-09-30T23:39:37.658+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:39:37.659+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:39:37.662+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:39:37.662+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:39:37.802+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:39:37.797+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:39:37.804+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:39:37.852+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.201 seconds
[2024-09-30T23:40:30.771+0000] {processor.py:186} INFO - Started process (PID=82) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:40:30.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:40:30.776+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:40:30.776+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:40:30.936+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:40:30.932+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:40:30.938+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:40:30.983+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.221 seconds
[2024-09-30T23:41:23.922+0000] {processor.py:186} INFO - Started process (PID=92) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:41:23.923+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:41:23.926+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:41:23.926+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:41:24.036+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:41:24.031+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:41:24.038+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:41:24.085+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.169 seconds
[2024-09-30T23:42:19.358+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:42:19.379+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:42:19.384+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:42:19.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:42:19.736+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:42:19.732+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:42:19.737+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:42:19.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.433 seconds
[2024-09-30T23:43:10.272+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:43:10.274+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:43:10.279+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:43:10.278+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:43:10.512+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:43:10.505+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:43:10.516+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:43:10.646+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.384 seconds
[2024-09-30T23:44:05.342+0000] {processor.py:186} INFO - Started process (PID=113) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:44:05.344+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:44:05.349+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:05.348+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:44:05.523+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:05.517+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:44:05.525+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:44:05.576+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.245 seconds
[2024-09-30T23:44:20.045+0000] {processor.py:186} INFO - Started process (PID=114) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:44:20.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:44:20.050+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:20.049+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:44:20.229+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:20.222+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Load.py", line 4, in <module>
    import airflow.dags.transformation as transformation
ModuleNotFoundError: No module named 'airflow.dags'
[2024-09-30T23:44:20.232+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:44:20.298+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.260 seconds
[2024-09-30T23:44:25.597+0000] {processor.py:186} INFO - Started process (PID=115) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:44:25.599+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:44:25.604+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:25.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:44:25.615+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:25.615+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/Load.py assumed to contain no DAGs. Skipping.
[2024-09-30T23:44:25.616+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:44:25.661+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.074 seconds
[2024-09-30T23:44:58.838+0000] {processor.py:186} INFO - Started process (PID=118) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:44:58.840+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:44:58.843+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:58.842+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:44:58.854+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:44:58.854+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/Load.py assumed to contain no DAGs. Skipping.
[2024-09-30T23:44:58.855+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:44:59.400+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.575 seconds
[2024-09-30T23:45:30.979+0000] {processor.py:186} INFO - Started process (PID=128) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:45:30.981+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:45:30.985+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:45:30.985+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:45:30.996+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:45:30.995+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/Load.py assumed to contain no DAGs. Skipping.
[2024-09-30T23:45:30.996+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:45:31.028+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.057 seconds
[2024-09-30T23:46:04.454+0000] {processor.py:186} INFO - Started process (PID=133) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:46:04.458+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:46:04.465+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:46:04.463+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:46:04.479+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:46:04.479+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/Load.py assumed to contain no DAGs. Skipping.
[2024-09-30T23:46:04.480+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:46:04.515+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.080 seconds
[2024-09-30T23:46:39.086+0000] {processor.py:186} INFO - Started process (PID=138) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:46:39.088+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:46:39.094+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:46:39.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:46:39.126+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:46:39.126+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/Load.py assumed to contain no DAGs. Skipping.
[2024-09-30T23:46:39.127+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:46:39.170+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.104 seconds
[2024-09-30T23:47:10.342+0000] {processor.py:186} INFO - Started process (PID=145) to work on /opt/airflow/dags/Load.py
[2024-09-30T23:47:10.344+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Load.py for tasks to queue
[2024-09-30T23:47:10.348+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:47:10.347+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Load.py
[2024-09-30T23:47:10.358+0000] {logging_mixin.py:190} INFO - [2024-09-30T23:47:10.358+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/Load.py assumed to contain no DAGs. Skipping.
[2024-09-30T23:47:10.359+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Load.py
[2024-09-30T23:47:10.408+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Load.py took 0.072 seconds
